lib/templates.rake

```diff
@@ -103,6 +107,23 @@ def update_snippet
   { :diff => diff, :status => status, :result => result }
 end
 
+def get_default_branch
+  # Always use develop on Foreman-nightly
+  return "-b develop" if Foreman::Version.new.tag == "develop"
+
+  url = URI.parse("https://api.github.com/repos/theforeman/community-templates/branches")
```

Comment:
I now see you already noticed this could be a problem. Still, could be nice to detect the branches for any type of repository, not just ours.